docs/style-guide.rst

```diff
@@ -112,7 +112,7 @@ No::
         }
     }
 
-Maximum Line Length
+Maximum Line Length _maximum_line_length:
```

Comment:
Has to be above the title and needs `..` prefix

Reply:
header is working now, `:ref` is still failing